### PR TITLE
Fix profiler for windows

### PR DESCRIPTION
**Pull request by mattcolegate**
Fixes https://github.com/eclipse/codewind/issues/2229

This PR allows the Java profiler to deal with Windows paths whilst preserving the working Unix path structure. It also contains fixes for the HotMethod scoping to accurately tally brackets to determine where class and method scope ends.

Tested manually in both Windows and Mac environments.

Please squash and merge.

### CODE REVIEW

File: client/src/extension.ts [hunk 1]
```diff
@@ -20,22 +24,31 @@ import { promisify } from 'util';
...
 import * as tarfs from 'tar-fs';
 import * as Docker from 'dockerode';
-import * as ip from 'ip';
 
 const docker = new Docker();
 
 const followProgress = promisify(docker.modem.followProgress);
 
 const clientPort: number = 3333;
+const clientHost: string = "host.docker.internal"
```
Comment (james-wallis) [inline client/src/extension.ts]: Single quotes

File: server/src/main/java/com/ibm/JavaProfilingLanguageServer/HotMethod.java [hunk 2]
```diff
@@ -89,10 +87,10 @@ public Range findInTextDocument(TextDocumentItem textDocument) {
...
 			contextRange.setEnd(new Position(contextRange.getStart().getLine(), startOfMethodName + endOfMethodName));
 
-			//		System.out.println("*** FINAL SCOPE ***");
-			//		System.out.printf("Range: start: %s:%s, end: %s:%s%n",
-			//				contextRange.getStart().getLine(), contextRange.getStart().getCharacter(),
-			//				contextRange.getEnd().getLine(), contextRange.getEnd().getCharacter());
+			// System.out.println("*** FINAL SCOPE ***");
+			// System.out.printf("Range: start: %s:%s, end: %s:%s%n",
+			// 	contextRange.getStart().getLine(), contextRange.getStart().getCharacter(),
+			// 	contextRange.getEnd().getLine(), contextRange.getEnd().getCharacter());
```
Comment (james-wallis) [inline server/src/main/java/com/ibm/JavaProfilingLanguageServer/HotMethod.java]: I know these were there before but shouldn't we be removing commented print lines?

File: server/src/main/java/com/ibm/JavaProfilingLanguageServer/HotMethod.java [hunk 3]
```diff
@@ -145,19 +143,29 @@ private String removeCommentsFromText(String text) {
...
+            count++;
+            fromIndex = foundIndex + 1;
+            foundIndex = inputString.indexOf(stringToMatch, fromIndex);
+        }
+        return count;
+    }
+
 	private Range getRangeInTextWithinScope(String scope, String text, Range range, int scopeDepth) {
-		// System.out.printf("scope: %s, text: %s, range: %s, scopDepth: %s", scope, text, range, scopeDepth);
+		// System.out.printf("scope: %s, text: %s, range: %s, scopeDepth: %s\n", scope, text, range, scopeDepth);
```
Comment (james-wallis) [inline server/src/main/java/com/ibm/JavaProfilingLanguageServer/HotMethod.java]: Same as above

File: server/src/main/java/com/ibm/JavaProfilingLanguageServer/ProfilingTextDocumentService.java [hunk 4]
```diff
@@ -154,7 +154,7 @@ private String getLatestLoadTestResults(String projectDir) {
 		List<Diagnostic> diagnostics = new ArrayList<Diagnostic>();
 
 		String documentPackage = getDocumentPackage(textDocument);
-//		System.out.println("documentPackage: " + documentPackage);
+		// System.out.println("documentPackage: " + documentPackage);
```
Comment (james-wallis) [inline server/src/main/java/com/ibm/JavaProfilingLanguageServer/ProfilingTextDocumentService.java]: same as above

File: client/src/extension.ts (hunk 1 above)
Comment (mattcolegate) [reply client/src/extension.ts]: Done - also added missing semi-colon

File: server/src/main/java/com/ibm/JavaProfilingLanguageServer/HotMethod.java (hunk 2 above)
Comment (mattcolegate) [reply server/src/main/java/com/ibm/JavaProfilingLanguageServer/HotMethod.java]: Removed

File: server/src/main/java/com/ibm/JavaProfilingLanguageServer/HotMethod.java (hunk 3 above)
Comment (mattcolegate) [reply server/src/main/java/com/ibm/JavaProfilingLanguageServer/HotMethod.java]: Removed

File: server/src/main/java/com/ibm/JavaProfilingLanguageServer/ProfilingTextDocumentService.java (hunk 4 above)
Comment (mattcolegate) [reply server/src/main/java/com/ibm/JavaProfilingLanguageServer/ProfilingTextDocumentService.java]: Removed

**Pull request merged**